app/src/main/java/org/oppia/app/drawer/NavigationDrawerItem.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.drawer
+
+import org.oppia.app.R
+
+/** Enum to store the menu of [NavigationDrawerFragment] and get menu by id. */
+enum class NavigationDrawerItem(val value: Int) {
+  HOME(R.id.nav_home), HELP(R.id.nav_help);
+
+  companion object {
+    fun valueFromInt(v: Int): NavigationDrawerItem {
+      var item = HOME
```

Comment:
Let’s avoid the var here. I think you can just return right on the when statement rather than relying on a variable.

Reply:
Done. updated to 
`  fun valueFromNavId(id: Int): NavigationDrawerItem {
      return when (id) {
        R.id.nav_home -> HOME
        R.id.nav_help -> HELP
        else -> HOME
      }
    }`